scripts/systems.py

```diff
@@ -232,6 +239,8 @@
 """,
 }
 
+cscs["alps-daint"] = cscs["alps-gh200"].copy()
+
```

Comment:
Just thinking out loud: should we either add
```suggestion
cscs["alps-daint"] = cscs["alps-gh200"].copy()
cscs["alps-clariden"] = cscs["alps-gh200"].copy()
cscs["alps-santis"] = cscs["alps-gh200"].copy()
```
or just remove this copy altogether:
```suggestion
```
? We can always just use `alps-gh200` as the "system" in scripts. Or is this copy meant as an example, saying "if you want to modify this, make a copy, and override options here"? I will most likely just modify `alps-gh200` directly locally instead (if I need to make changes at all), but not everyone has to do that.

Reply:
The idea was to have the ability to have different presets for the different clusters, so the copy was just there, as you said, as example.

But I agree, I did the same thought as you: currently most of the things that I would change are in a blob (the string with the header of the batch script), and it is more convenient to change that in-place or anyway replace it completely.

In addition to the fact that this file act just as a reference and it is open to customisation and changes for the different use-cases, one thing I always (forget to) add is the `--uenv`, and I add it in-place modifying the original one.

So, I agree that we can keep just `alps-gh200` and change inline whatever is needed. In case in the future there will be more intricate use-cases, we will decide how to deal with that.